NLog.Web.AspNetCore/NLog.Web.AspNetCore.csproj

```diff
@@ -30,10 +30,10 @@ Supported platforms:
     <PackageId>NLog.Web.AspNetCore</PackageId>
     <PackageTags>logging;log;session;NLog;web;aspnet;aspnetcore;MVC;httpcontext</PackageTags>
     <PackageReleaseNotes>
-- Added ThreadSafe-attribute for LayoutRenderer to optimize async Precalculate (@snakefoot)
-- ${AspNetRequestIp} added CheckForwardedForHeader to check for X-Forwarded-For header (#325) (@Giorgi)
-- Check that Content-Type is correct before accessing the Form (#322) (@mcliment)
-- Enabled SymbolPackageFormat=snupkg (@snakefoot)
+- Updated Microsoft.AspNetCore to ver. 2.1 (Long-Term-Support)
+- ${configsetting} layout renderer now depends on UseNLog-call to provide IConfiguration using IServiceProvider
+- NLogLoggerProvider options now binds to IConfiguration (Using section Logging:NLog)
+- UseNLog-call now also available for IHostBuilder
```

Comment:
this one isn't released yet: https://github.com/NLog/NLog.Web/milestone/29?closed=1